packages/core-components/src/layout/ErrorPage/MicDrop.tsx

```diff
@@ -28,7 +28,7 @@ const useStyles = makeStyles(
         maxWidth: '96%',
         bottom: 'unset',
         right: 'unset',
-        margin: `${theme.spacing(10)}px auto ${theme.spacing(4)}px`,
+        margin: theme.spacing(10, 'auto', 4),
```

Comment:
is this going to do exactly what we expect it to in both versions of mui @tudi2d with the helper?